modules/op-changing-default-service-account.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-changing-default-service-account_{context}"]
+= Changing the default service account
+
+You can change the default service account by editing the `default-service-account` field in the `.spec.pipeline` and `.spec.trigger` specifications. The default service account name is `pipeline`.
```

Comment:
The default service account for what? This module doesn't really make it clear what the default service account is changing for, and SAs are used for many functions in OCP. I would clarify this.